keps/sig-multicluster/1645-multi-cluster-services-api/README.md

```diff
@@ -538,6 +538,11 @@ cluster, non-cluster-admin users should not be allowed to create or modify
 `ServiceImport` resources. The mcs-controller should be solely responsible for
 the lifecycle of a `ServiceImport`.
 
+Some errors may occur during the `ServiceImport`'s lifecycle, such as IP protocol
```

Comment:
this new use case may impact the discussion on the [the serviceImport api](https://github.com/kubernetes-sigs/mcs-api/issues/106).